src/microbe_stage/PatchManager.cs

```diff
@@ -245,6 +258,17 @@ private void ClearUnmarkedSingle(List<CreatedSpawner> spawners)
         });
     }
 
+    private void QueueUnmarkedForDestruction(List<CreatedSpawner> spawners)
+    {
+        foreach (CreatedSpawner spawner in spawners)
+        {
+            if (!spawner.Marked)
+            {
+                spawner.Spawner.DestroyQueued = true;
```

Comment:
I think that this could be merged with the RemoveNonMarkedSpawners, there's that one print statement anyway that triggers with the same condition, that way there's less of a chance of accidentally removing a spawner that wasn't queued for destruction.

Reply:
Yeah no problem, that is how I originally had it (in the first commit there) but was undecided on the best approach. I will change it back.

Reply:
Should fit the requirement now.